task1.js

```diff
@@ -0,0 +1,119 @@
+const express = require('express');
+const bp = require('body-parser');
+const Joi = require('joi');
+const app=express();
+const port=3000;
+
+app.use(bp.json());
+
+const user1={
+  id: '1',
+  login: 'us1',
+  password: 'us1',
+  age: 6,
+  isDeleted: false,
+};
+const user2={
+  id: '2',
+  login: 'us2',
+  password: 'us2',
+  age: 13,
+  isDeleted: false,
+};
+const users=[user1, user2];
+
+app.get('/', (req, res) => {
```

Comment:
Use multilayer architecture for this home work.
Possible structure of project is:
api/user/
|- controller - validate input, call service, and create a response from service results
|- service - manage the logic of input processing and calls repo
|- repository - contains modules to work with repositories (CSV file in your case)
|- router.js - store routes and routes handlers for entity user

package.json
.....